### ISSUE TITLE
Prev/Next links on story pages point to deleted stories

### ISSUE THREAD

**Issue by scotthansonde** [scripting/1999-project#20]
The new delete function will remove deleted stories from the home page and the rss feed, and produces an empty story page for the deleted story. However, the Prev/Next links (the green arrows to the left and right of the story) on the story pages before and after the deleted story will point to the empty page for the deleted story.

For example, the first story for my my.1999.io site is http://my.1999.io/users/papascott/2016/04/15/0001.html. 

The "Next" (the green arrow to the right) points to http://my.1999.io/users/papascott/2016/04/19/0004.html, which is a story I've deleted.

I guess I'd expect the link to lead to the next published story.

**Comment by scripting** [comment]
This should be fixed. It required both a server and an editor release.

**Comment by scotthansonde** [comment]
It's not quite working for me yet...
Created a new site. Published posts "One", "Two" and "Three". 
I deleted "Two". "Three" still points back to "Two".
I republish all pages. "Three" still points back to "Two".
I edit "Three" from the editor page. "Three" still points back to "Two".
I edit "Three" from the story page. Now "Three" points back to "One", which is correct.
~~This was just casual testing... I didn't check, for example, if it was just my browser cache not updating. I'll get a chance later to try this more rigorously.~~~
Update May 10: I confirmed this behavior by examining the value of prev.id in the pagetable in the html files directly. Version 0.94m/0.89g.
